public/css/homepage.css

```diff
@@ -1,3 +1,18 @@
+body {
```

Comment:
Homepage.css is added to `.gitignore` but that does not take effect. I think it's windows-ic issue.

Reply:
I guess you also need to delete file and make another commit with deleted file, as it's being tracked since it was added first time.

Reply:
`git rm --cached public/css/homepage.css` resolved this I guess.